.github/workflows/create-release.yml

```diff
@@ -175,12 +175,20 @@ jobs:
         cp install_sct.bat install_sct-${{ github.event.inputs.milestone_title }}_win.bat
         perl -pi -e 's/set git_ref=master/set git_ref=${{ github.event.inputs.milestone_title }}/' install_sct-${{ github.event.inputs.milestone_title }}_win.bat
 
+    - name: Create ${{ github.event.inputs.milestone_title }} Linux/macOS install scripts
+      # NB: `install_sct` works for both Linux and macOS platforms. So, here, we simply duplicate the script for both
+      # platforms (which will allow us to collect some rudimentary statistics + make our installation steps clearer).
+      run: |
+        cp install_sct install_sct-${{ github.event.inputs.milestone_title }}_linux.sh
```

Comment:
When renaming the file, I chose to add the `.sh` file extension, largely for symmetry with the `install_sct-6.0_win.bat` script. (I've always found it a little weird that we chose to drop the file extension for `install_sct`?)

I wanted to point this out so that we maybe discuss/brainstorm the consequences of this change? (Were there benefits to excluding the extension in the past?)

Reply:
As I understand it, from sources like ["command names considered harmful"](https://www.talisman.org/~erlkonig/documents/commandname-extensions-considered-harmful/), it's typical Unix practice to not use file extensions for any executables that live on the `$PATH`, because unlike Windows, file extensions are not used to determine how to run the file (rather that's the job of the shebang line inside the file).

But! There's some nuance here: this is Unix-specific, and it's only for things on the `$PATH`. For an example of another case, the [Google style guide](https://google.github.io/styleguide/shellguide.html#s2.1-file-extensions) recommends using extensions for shell *libraries* (that are sourced from other shell scripts with `.` or `source`, so that they can declare functions and modify variables).

In our case, I would argue that:
- We're in a non-Unix-specific context (the release download page, right next to a Windows executable), so the extension is helpful to the reader.
- We're *not* on the `$PATH`, and after downloading the file, it might not even have executable permissions. So users might have to either `chmod +x install_sct` or invoke it through the shell, like `bash install_sct`.
- We might even want to use `.bash` instead of `.sh`, because our script probably fails if it's run with `sh install_sct` instead of `bash install_sct`.
- Probably for all these reasons, I think specifically for *installers* it's common to put the `.sh` extension.

Reply:
> * We might even want to use `.bash` instead of `.sh`, because our script probably fails if it's run with `sh install_sct` instead of `bash install_sct`.

This is a really good point!

Though, from what I understand, `install_sct` makes a point to support non-bash shells (e.g. `zsh`), especially since macOS [defaults to `zsh`](https://www.theverge.com/2019/6/4/18651872/apple-macos-catalina-zsh-bash-shell-replacement-features) these days? And, I'm slightly concerned that a `.bash` extension may scare away/confuse macOS users who don't understand the nuances of shell types. 

`.sh` may be inaccurate, but it signals that it's a shell-agnostic script to some degree, I think?